resources/assets/js/markdown-editor/css/style.css

```diff
@@ -0,0 +1,1380 @@
+/* purgecss start ignore */
```

Comment:
does this contain any custom css or is it a copy of the editor's default css? if it's a copy, can we not link to the original css instead of copying it here (error prone in case of updates) and whitelist `.tui` in purgecss?

Reply:
@ItsANameToo Is a modified copy of the default editor it has a lot of changes mostly related to the toolbar is not easy to guess what is used and not only through trial and error, on Monday ill try to clean up it a bit

Reply:
@alfonsobries if it's a copy with adjustments, would it be possible to include the original CSS and then overwrite it by defining the changes we need in a separate file? so we only end up with a file containing the CSS changes and the rest will remain in the original CSS that we can simply include (makes it a lot easier when dealing with updates)